.github/workflows/comment-run.yml

```diff
@@ -0,0 +1,43 @@
+# This job is to test different maven profiles in sdk branch again Pull Request raised
+# This workflow targets Java with Maven execution
+
+name: TestNG SDK Test workflow for Maven on comment RUN_TESTS
+
+on:
+  issue_comment:
+    types: [ created, edited ]
+
+jobs:
+  comment-run:
+    if: contains(github.event.comment.body, 'RUN_TESTS')
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      max-parallel: 3
+      matrix:
+        java: [ '8', '11', '17' ]
+        os: [ 'macos-latest', 'windows-latest', 'ubuntu-latest' ]
+    name: TestNG Repo ${{ matrix.Java }} - ${{ matrix.os }} Sample
+    env:
+      BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+      BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
+      - name: Run mvn test
+        run: |
+          mvn compile
+          mvn test
```

Comment:
which profile will this use.. (just checking if nor redundant with the two added below)

Reply:
resolved f2f, is a run command on documentation